### DISCUSSION TITLE
SMS authentication not available for Ethiopia

### DISCUSSION THREAD

**Discussion by DemisoDaba** [community/community#123424] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Dear GitHub community, SMS authentication not available for Ethiopia, How can we Ethiopians will Authenticate our account?

**Comment by CiceroLino** [comment]
Can you just use email to authentication?

**Reply by DemisoDaba** [reply]
![image](https://github.com/community/community/assets/125874545/11a4d096-6ba9-47e8-9f4f-93843bd3dbb6)
![image](https://github.com/community/community/assets/125874545/4f743ab3-c63c-4a4d-a328-ebe37236e33f)
no Ethiopia in Country code list. pls add ethiopia to the list and i can use my phone directly. there is no option to verfy by email. it is 2fa just app and sms
